platform/ui-next/src/components/Thumbnail/Thumbnail.tsx

```diff
@@ -5,6 +5,8 @@ import { useDrag } from 'react-dnd';
 import { Icons } from '../Icons';
 import { DisplaySetMessageListTooltip } from '../DisplaySetMessageListTooltip';
 import { TooltipTrigger, TooltipContent, Tooltip } from '../Tooltip';
+import { useCustomization } from '@ohif/core';
+import { CSSStylesCustomization } from '@ohif/core/src/services/CustomizationService/types';
```

Comment:
I'm not really sure about this. I like the idea of a `useCustomization` hook, but importing from `ohif/core` isn't ideal. If you search for `ohif/core` imports in the `ui-next` folder, you'll probably find none, or if there are any, they're really necessary.

Also, I don't think you need this customization to apply this logic. You could have listeners for the `useCustomization` changes and just target the DOM elements to apply your styles. Instead of that, we're introducing a whole other naming schema for customizing bits and pieces of the UI. I mean, you can use `querySelector` to select the element you want to customize.

Reply:
I'd rather add a unique ID to each element through the app. I don't mind you submitting pull requests to add proper IDs to each of our UI elements. Then, you can place the `useCustomization` hook in your extension/mode, allowing you to target each element and apply your style.

Basically, we have two options:

1.  Add `useCustomization` to each UI element and then target it using each ID, which isn't documented anywhere.
2.  Add an ID to each UI element.

I favor the second method because it doesn't tie the `useCustomization` implementation to the UI.